WordPress/Classes/System/SplitViewRootPresenter.swift

```diff
@@ -301,6 +309,14 @@ final class SplitViewRootPresenter: RootViewPresenter {
     }
 }
 
+extension SplitViewRootPresenter: UISplitViewControllerDelegate {
+    func splitViewController(_ svc: UISplitViewController, willHide column: UISplitViewController.Column) {
+        if column == .primary {
+            sitePickerPopoverVC?.presentingViewController?.dismiss(animated: true)
```

Comment:
Not exactly sure what the difference is, but is `sitePickerPopoverVC.dismiss()` sufficient here?

Reply:
There isn't any difference in the current setup, but I never use `dismiss` on the presented view controller because it will fail if it itself is presenting another view controller. The safest option is to access `presentingViewController` to check if the screen is currently being presented and then ask it directly to dismiss its presented view controller, which is what this code does.

"The presenting view controller is responsible for dismissing the view controller it presented. If you call this method on the presented view controller itself, UIKit asks the presenting view controller to handle the dismissal."